$OpenBSD: patch-Recipe,v 1.1 2019/03/31 10:55:48 sthen Exp $

From f5c1753244d0399764fd68d6ac42ea2f2db47fc9 Mon Sep 17 00:00:00 2001
From: Simon Tatham <anakin@pobox.com>
Date: Tue, 26 Mar 2019 19:12:27 +0000
Subject: [PATCH] Link uxutils.o into Unix PuTTYgen.

On Arm Linux, this is necessary for the functions that check
availability of hardware crypto acceleration.

Index: Recipe
--- Recipe.orig
+++ Recipe
@@ -365,7 +365,7 @@ PUTTYGEN_UNIX = sshrsag sshdssg sshprime sshdes ARITH 
          + sshrand uxnoise sshsha MISC sshrsa sshdss uxcons uxstore uxmisc
          + sshpubk sshaes sshsh256 sshsh512 IMPORT puttygen.res time tree234
          + uxgen notiming CONF sshecc sshecdsag uxnogtk sshauxcrypt sshhmac
-         + uxpoll
+         + uxpoll uxutils
 puttygen : [U] cmdgen PUTTYGEN_UNIX
 cgtest   : [UT] cgtest PUTTYGEN_UNIX
 
